### PR TITLE
fix(components): resort attributelist after delete result item

### PR DESCRIPTION
**Pull request by gary-Shen**
Fix
- [4135873](https://project.feishu.cn/bigdata_03/issue/detail/4135873?parentUrl=%2Fbigdata_03%2FissueView%2FXARIG5p4g)
- [4136011](https://project.feishu.cn/bigdata_03/issue/detail/4136011?parentUrl=%2Fbigdata_03%2FissueView%2FXARIG5p4g)

### CODE REVIEW

File: packages/components/src/views/MainView/rightSiderBar/AttributeRusult/index.tsx [hunk 1]
```diff
@@ -199,74 +199,66 @@ const AttributeRusult: FC<IProps> = ({
   }, [copyToolInstance, currentToolName, imgIndex, imgList]);
 
   useEffect(() => {
-    if (imgList && imgList.length > 0 && imgList.length > imgIndex) {
-      const currentImgResult = JSON.parse(imgList[imgIndex].result as string);
```
[the remark this answers is not in the document]
Comment (gary-Shen) [reply packages/components/src/views/MainView/rightSiderBar/AttributeRusult/index.tsx:203]: 把原来的条件嵌套重构了一下，影响bug的改动就加了个sortBy.

[the remark this answers is not in the document]
Comment (wuhui1989) [reply packages/components/src/views/MainView/rightSiderBar/AttributeRusult/index.tsx:203]: ok

**Review by wuhui1989 — APPROVED** [review]

**Pull request merged**